### PR TITLE
Adding comparison tables

### LINKED ISSUE #137: Compare run time and accuracy in one place with colours
Feedback from Tyrone:

He would like there to be a way for the user to have an indication of the accuracy and run time in one place. A suggestion was to use the outline colours of the runtime cells on an accuracy table and visa.

Further implement this to be an user option, i.e. something which the user can turn off and on, implemented after #167 is completed.

### PR DESCRIPTION
**Pull request by wathen**
#### Description of Work

Fixes #137 

Adds additional table which compares runtime and accuracy in one place. 

#### Testing Instructions

1. Check combined colours are consistent with runtime and accuracy separately 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

File: fitbenchmarking/utils/fitbm_result.py [hunk 1]
```diff
@@ -43,12 +44,18 @@ def __init__(self, options=None, problem=None, fit_status=None,
...
     def __str__(self):
         if self.table_type is not None:
-            if self.options.comparison_mode == "abs":
-                output = "{:.4g}".format(self.value)
-            elif self.options.comparison_mode == "rel":
-                output = "{:.4g}".format(self.value)
-            elif self.options.comparison_mode == "both":
-                output = "{:.4g} ({:.4g})".format(self.value, self.norm_value)
+            if self.table_type == "compare":
+                output = "{:.4g} ({:.4g})<br>{:.4g} ({:.4g})".format(
```
[the remark this answers is not in the document]
Comment (wathen) [reply fitbenchmarking/utils/fitbm_result.py]: Has been corrected

File: fitbenchmarking/utils/fitbm_result.py [hunk 2]
```diff
@@ -43,12 +44,18 @@ def __init__(self, options=None, problem=None, fit_status=None,
...
+                    self.chi_sq, self.norm_acc,
+                    self.runtime, self.norm_runtime)
+            else:
+                if self.options.comparison_mode == "abs":
+                    output = "{:.4g}".format(self.value)
+                elif self.options.comparison_mode == "rel":
+                    output = "{:.4g}".format(self.value)
```
[the remark this answers is not in the document]
Comment (wathen) [reply fitbenchmarking/utils/fitbm_result.py]: done

**Comment by wathen** [comment]
I have added co

> Looks good.
> Did you want me to merge this and create an issue for adding it as an option, or should we do that on this PR?
> The benefit of adding it to options means we can add docs explaining the tables (as there's no obvious place to do that at the moment).

I have added these to the options file now and correct the path to the default options in the docs

File: fitbenchmarking/utils/tests/test_options.py [hunk 3]
```diff
@@ -68,11 +71,12 @@ def test_from_file(self):
         self.assertEqual(fitting_opts['use_errors'], options.use_errors)
         self.assertEqual(fitting_opts['num_runs'], options.num_runs)
         self.assertEqual(fitting_opts['software'], options.software)
-
+        print(options.table_type)
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/utils/tests/test_options.py]: Left over print?

File: fitbenchmarking/utils/default_options.ini [hunk 4]
```diff
@@ -109,6 +109,21 @@ colour_scale: 1.1, #fef0d9
...
+#                 options are 'acc', 'runtime', 'compare'
+#                 'acc' indicates that the resulting table should contain the
+#                       chi_sq values for each of the minimizers
+#                 'runtime' indicates that the resulting table should contain
+#                           the runtime values for each of the minimizers
+#                 'compare' indicates that the resulting table should contain
+#                           both the chi_sq value and runtime values for each
+#                           of the minimizers. The tables produced have the
+#                           chi_sq values on the top line of the cell and the
+#                           runtime on the bottom line of the cell.
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/utils/default_options.ini]: Can you insert a `# default is 'acc', 'runtime', and 'compare'`.
Then we need to copy this to the examples dir, maybe we need to come up with a better way of keeping these up to date with each other.

Comment (wathen) [reply fitbenchmarking/utils/default_options.ini]: done

File: fitbenchmarking/utils/tests/test_options.py (hunk 3 above)
Comment (wathen) [reply fitbenchmarking/utils/tests/test_options.py]: Corrected

**Pull request merged**